samples/v2/get-instance-config.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//  title: Gets the instance config metadata for the configuration nam6
+//  usage: node get-instance-config.js <PROJECT_ID> <INSTANCE_CONFIG_ID>
+
+'use strict';
+
+function main(projectID) {
+  // [START spanner_get_instance_config]
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = 'my-project-id';
+
+  // Imports the Google Cloud client library
+  const {InstanceAdminClient} = require('@google-cloud/spanner/build/src/v1');
+
+  // Creates a client
+  const instanceAdminClient = new InstanceAdminClient({
```

Comment:
Also based on discussion https://github.com/googleapis/python-spanner/pull/1065#discussion_r1461941195

We need to define a strategy for how this should work with the emulator. Currently, we tell our customers that they can just set the environment variable `SPANNER_EMULATOR_HOST=localhost:9010` and everything will use the emulator. That is no longer true with this setup. We should be consistent across all languages regarding this, both to ease our documentation job, but also to make it easier for customers who do use multiple programming languages.
The simplest solution is probably to instruct customers to create a 'normal' hand-written client, and then get the auto-generated admin client from that hand-written client. That allows us to return the same auto-generated admin client that we would otherwise have used in combination with the hand-written client.